js_modules/dagster-ui/packages/ui-core/src/assets/useAssetDefinitionFilterState.oss.tsx

```diff
@@ -224,3 +224,22 @@ export function filterAssetDefinition(
 
   return true;
 }
+
+export function getAssetFilterStateQueryString() {
+  const values: Record<keyof AssetFilterType, '1'> = {
```

Comment:
Using a Record here as a way to make sure that if we add any new filters that we raise a typescript error if theyre not added here